nuke fails with missing file

nuke fails if your repo isn't fully bootstrapped before nuking with `rm: index.js: No such file or directory` in 'react-server-middleware-json-response'.

```
» npm run nuke && npm run bootstrap && npm test | tee nuke-test.out                                                                        130 ↵

> @ nuke /Users/dwade/foss/react-server
> npm run clean && asini clean && rm -r node_modules


> @ clean /Users/dwade/foss/react-server
> rimraf asini-debug.log && asini run clean


> react-server-middleware-json-response@0.4.13 clean /Users/dwade/foss/react-server/packages/react-server-middleware-json-response
> rm index.js


NpmUtilities.execInDir        ("run clean", [], "./packages/react-server-middleware-json-response")

> react-server-middleware-json-response@0.4.13 clean /Users/dwade/foss/react-server/packages/react-server-middleware-json-response
> rm index.js


NpmUtilities.runScriptInDir   ("clean", [], "./packages/react-server-middleware-json-response")

> react-server-middleware-json-response@0.4.13 clean /Users/dwade/foss/react-server/packages/react-server-middleware-json-response
> rm index.js


Errored while running npm script 'clean' in 'react-server-middleware-json-response'
Error: Command failed: npm run clean
rm: index.js: No such file or directory

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "clean"
npm ERR! node v6.9.4
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! react-server-middleware-json-response@0.4.13 clean: `rm index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-server-middleware-json-response@0.4.13 clean script 'rm index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-server-middleware-json-response package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-server-middleware-json-response
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-server-middleware-json-response
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/packages/react-server-middleware-json-response/npm-debug.log

    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
Errored while running RunCommand.execute
Error: Command failed: npm run clean
rm: index.js: No such file or directory

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "clean"
npm ERR! node v6.9.4
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! react-server-middleware-json-response@0.4.13 clean: `rm index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-server-middleware-json-response@0.4.13 clean script 'rm index.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-server-middleware-json-response package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-server-middleware-json-response
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-server-middleware-json-response
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/packages/react-server-middleware-json-response/npm-debug.log

    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)

> react-server-module-tagger@0.4.10 clean /Users/dwade/foss/react-server/packages/react-server-module-tagger
> del npm-debug.log*


NpmUtilities.execInDir        ("run clean", [], "./packages/react-server-module-tagger")

> react-server-module-tagger@0.4.10 clean /Users/dwade/foss/react-server/packages/react-server-module-tagger
> del npm-debug.log*


NpmUtilities.runScriptInDir   ("clean", [], "./packages/react-server-module-tagger")

> react-server-module-tagger@0.4.10 clean /Users/dwade/foss/react-server/packages/react-server-module-tagger
> del npm-debug.log*


Errored while running npm script 'clean' in 'react-server-module-tagger'
Error: Command failed: npm run clean
sh: del: command not found

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "clean"
npm ERR! node v6.9.4
npm ERR! npm  v4.1.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! react-server-module-tagger@0.4.10 clean: `del npm-debug.log*`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the react-server-module-tagger@0.4.10 clean script 'del npm-debug.log*'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-server-module-tagger package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     del npm-debug.log*
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-server-module-tagger
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-server-module-tagger
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/packages/react-server-module-tagger/npm-debug.log

    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:498:12)
Waiting for 3 child processes to exit. CTRL-C to exit immediately.

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/foss/react-server/node_modules/.bin/npm" "run" "clean"
npm ERR! node v6.9.4
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! @ clean: `rimraf asini-debug.log && asini run clean`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ clean script 'rimraf asini-debug.log && asini run clean'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rimraf asini-debug.log && asini run clean
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/npm-debug.log

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/dwade/n/bin/node" "/Users/dwade/n/bin/npm" "run" "nuke"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! @ nuke: `npm run clean && asini clean && rm -r node_modules`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ nuke script 'npm run clean && asini clean && rm -r node_modules'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run clean && asini clean && rm -r node_modules
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dwade/foss/react-server/npm-debug.log
```

--- a/packages/react-server-middleware-json-response/package.json
+++ b/packages/react-server-middleware-json-response/package.json
@@ -6,7 +6,7 @@
   "scripts": {
     "prepublish": "npm run build",
     "build": "babel index.src.js --out-file index.js",
-    "clean": "rm index.js",
+    "clean": "rimraf index.js",
     "test": "eslint src"
   },
   "repository": {
@@ -31,6 +31,7 @@
     "babel-preset-react-server": "^0.6.0",
     "eslint": "^3.8.1",
     "eslint-plugin-react": "^6.4.1",
-    "babel-eslint": "^7.0.0"
+    "babel-eslint": "^7.0.0",
+    "rimraf": "^2.5.4"
   }
 }

